docs/build-module-cr.md

```diff
@@ -0,0 +1,34 @@
+| [Home](../README.md) |
+|----------------------|
+
+## CR for Building a new Module
+
+This task entails following sub-tasks:
+
+1. Application administrator creates a new CR for building a new module. For creating a new CR refer to the [New Change Request from Production Environment](#new-change-request-from-production-environment).
+
+2. Content Developer creates a new module. For creating a new module, refer to the article [Creating a New Module](https://docs.fortinet.com/document/fortisoar/7.3.1/administration-guide/97786/application-editor#Creating_a_New_Module).
+
+3. Content developer adds the new module to the navigation menu. Refer to this article for [Modifying the Navigation Bar](https://docs.fortinet.com/document/fortisoar/7.3.1/administration-guide/97786/application-editor#Modifying_the_Navigation_bar).
+
+4. Depending on the module and requirements, content developer may need to create new roles, or edit existing roles to assign permissions for the new module.
+
+    1. For creating new roles, refer to the article [Adding Roles](https://docs.fortinet.com/document/fortisoar/7.3.1/administration-guide/202940/security-management#Adding_Roles).
+
+    2. For editing permissions assigned to roles, refer to the article [Configuring Roles](https://docs.fortinet.com/document/fortisoar/7.3.1/administration-guide/202940/security-management#Configuring_Roles).
+
+5. Content developer edits the **Source Control &ndash; Production Content** template to include module, roles, picklists, and any other content created to support the new module.
```

Comment:
Content developer edits the **Source Control &ndash; Production Content** template by going to "Source Control Settings" card and include module, roles, picklists, and any other content created to support the new module.

Reply:
Fixed